pymc_experimental/preprocessing/standard_scaler.py

```diff
@@ -0,0 +1,16 @@
+import pandas as pd
+from sklearn.base import BaseEstimator, TransformerMixin
+from sklearn.preprocessing import StandardScaler
+
+
+class StandardScalerDF(StandardScaler, TransformerMixin, BaseEstimator):
```

Comment:
Is this only for dataframe support? Because I think sklearn added that: https://blog.scikit-learn.org/technical/pandas-dataframe-output-for-sklearn-transformer/

Reply:
Also https://ponder.io/scikit-learns-transformers-now-output-pandas-dataframes/

Reply:
thanks! Indeed I've added that as I've noticed that It was always returning ndarray, which was causing us to loose info about column names, and that was not really acceptable. I'll include the configurable sklearn solution instead of that

Reply:
But then you can just remove this code here, no?